dist/21-686C-schema.json

```diff
@@ -2962,9 +2959,6 @@
     "veteranSocialSecurityNumber": {
       "$ref": "#/definitions/ssn"
     },
-    "claimantSocialSecurityNumber": {
```

Comment:
Are these two deletions from this file something that should have been done in a previous PR?

Reply:
Not necessarily.  We decided that the claimant will always be the veteran, so no need to have both `claimantSocialSecurityNumber ` and `veteranSocialSecurityNumber ` on the form.